plugins/mcreator-core/datalists/achievements.yaml

```diff
@@ -76,21 +77,27 @@
 - adventure/fall_from_world_height
 - adventure/play_jukebox_in_meadows
 - adventure/trade_at_world_height
+- adventure/kill_mob_near_sculk_catalyst
 - husbandry/root
-- husbandry/safely_harvest_honey

```

Comment:
Reordering belongs in a separate PR as it makes the review process rather tedious. Making it a separate commit would also work in a way.